my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
+import os
+from astropy.io import fits
+import glob
+data = fits.getdata('/Users/bostroem/Desktop/images/hubble_img.fits')
+col1 = data[20, :] 
+col2 = data[200, :]
+col3 = data[800, :]
+pyplot.figure()
+pyplot.plot(np.arange(1024), col1)
+mean = np.mean(col1)
+pyplot.axhline(mean)
+pyplot.legend(['Column 20', 'Mean'])
+pyplot.title('Plot of Column 20 Values and Mean')
+pyplot.xlabel('Pixels')
+pyplot.ylabel('Intensity')
+pyplot.figure()
```

Comment:
It looks like you're making several figures throughout the code. You could name them like "fig1 = pyplot.figure()" (bonus if the name is more descriptive than just fig1!) and add whitespace (a blank line) between each plot to make it clearer which commands go with which plot.